shoop/core/models/addresses.py

```diff
@@ -173,6 +136,46 @@ def as_string_list(self, locale=None):
     def __iter__(self):
         return iter(self.as_string_list())
 
+    def immutable(self):
```

Comment:
I think this should be `get_immutable()`, or `as_immutable()`, or similar, so that methods remain verby (though I'm aware "as" is not a verb but a preposition here).